src/aplRuntime.ts

```diff
@@ -316,6 +316,7 @@ export class AplRuntime extends EventEmitter {
 			case 'CloseWindow': this.closeWindow(rideMessage[1] as CloseWindowMessage); break;
 			case 'Disconnect': this.disconnect(rideMessage[1] as DisconnectMessage); break;
 			case 'EchoInput': this.echoInput(rideMessage[1] as EchoInputMessage); break;
+			case 'GetHelpInformation': this.getHelpInformation(rideMessage[1] as GetHelpInformationMessage); break;
```

Comment:
The interpreter will never send a GetHelpInformation to RIDE client. Remove this line.